css/style.css

```diff
@@ -8,73 +8,150 @@ html {
   box-sizing: inherit;
 }
 
+body {
+  margin: 0 auto;
+}
+
+h1,
+h2 {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+ul {
+  list-style-type: none;
+}
+
 .main-container {
-  font-family: Inter, sans-serif;
   display: flex;
   flex-direction: column;
   align-items: center;
+  font-family: Inter, sans-serif;
```

Comment:
- let's make sure we get rid of unwanted space at the footer to match the Figma design

Your implementation | Figma Design
:------:|:-----:
![image](https://user-images.githubusercontent.com/87689547/160120864-811d13d9-97a8-4bce-9a17-d5d71abf88f1.png) | ![image](https://user-images.githubusercontent.com/87689547/160121014-1745de45-f57a-4083-92a6-5508db1403b5.png)

Reply:
Thanks for the comments, I have fixed all the issues.